use-cases/org-existing-cloudtrail-ecs-vpc-subnet.md

```diff
@@ -1,41 +1,35 @@
-# OrganizationSetup - Existing Cloudtrail - Existing ECS/VPC/Subnet
+# AWS Organizational Setup with Cloudtrail
```

Comment:
suggest to change i to
# AWS Organizational Setup with existing ECS/VPC/Subnet 

since that's the main interesting point to cover here

Reply:
improved

Reply:
3 resources are required together, ECS/VCP/Subnet.
can't do one without the other